hpy/devel/include/hpy.h

```diff
@@ -100,6 +100,7 @@ typedef struct { intptr_t _i; } HPyTracker;
 #define HPy_IsNull(h) ((h)._i == 0)
 
 #define HPyField_NULL ((HPyField){0})
+#define HPyField_IsNull(f) ((f)._i == 0)
```

Comment:
I would actually not like to expose `HPyField_IsNull`  but since `HPy_VISIT` is public, there are not many options. Anyway, I think it's not a problem since we already expose `HPyField_NULL` and so already implicitly specify that uninitialized `HPyField` has value `0`.

Reply:
This macro is identical to `HPy_IsNull`, I wonder if we should turn them both into static inline functions to prevent confusions.

Reply:
IIRC we had a comment in an older version of `hpy.c` that also asked this question and since `HPy_FromVoidP` and `HPy_AsVoidP` are also inline functions, it would make sense to me.
Let me merge this PR and I'll open a separate PR to do that. Then we can invite people to discuss that.